clmm/theory/ccl.py

```diff
@@ -105,6 +112,7 @@ def _get_mass(self):
     def _set_concentration(self, cdelta):
         """"set concentration. Also sets/updates hdpm"""
         self.conc = ccl.halos.ConcentrationConstant(c=cdelta, mdef=self.mdef)
+        ccl.UnlockInstance.Funlock(type(self.mdef), "_concentration_init", True)
         self.mdef._concentration_init(self.conc)
```

Comment:
Calling `UnlockInstance` and `_concentration_init` requires CCL between 2.7.1.dev1 and 2.7.1.dev9. Putting a if statement can extent the range to 2.6 to 2.7.1.dev9.

Reply:
Between 2.7.1.dev1 and 2.7.1.dev9 there is not conda installable versions, so that is why I hesitated to use only dev versions.

Reply:
please add it